docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md

```diff
@@ -81,27 +97,76 @@ The Platform configuration loader tries to fetch the configuration from up to th
+
+Note: secrets will always start with 'SECRET'
+
+```text
+# REQUIRED - The secret key of JWT encryption
+SECRET_SEP10_JWT_SECRET=<secret>
+
+# REQUIRED - The private key of the SEP-10 challenge.
+# We highly recommend that this private key should not be used to sign any transactions to submit to the Stellar
+# network.
+SECRET_SEP10_SIGNING_SEED=<secret>
+
+# REQUIRED - JWT secrets used to communicate between Anchor and Platform.
+SECRET_CALLBACK_API_AUTH_SECRET=<secret>
+SECRET_PLATFORM_API_AUTH_SECRET=<secret>
+```
+
+
+
+### Authorization Between Platform and Anchor
 
-It's possible to enable/disable authorization headers for requests between Platform and Anchor by editing the `integration-auth` configuration in the Platform config map. You can use different secrets depending on the direction of the requests, i.e. one for `Platform->Anchor` and another for `Anchor->Platform`, and you can choose between the following auth options:
-- `JWT_TOKEN`: where a secret is used to create a jwt token in the sender side, and this same secret is used to decode the token in the receiver side. This token is added to the `Authorization` header.
+It's possible to enable/disable authorization headers for requests between Platform and Anchor by editing the 
+`auth` configuration in the Platform config map. You can use different secrets depending on the direction 
+of the requests, i.e. one for `Platform->Anchor` and another for `Anchor->Platform`, and you can choose between the 
+following `auth.type` options:
+- `JWT_TOKEN`: where a secret is used to create a jwt token in the sender side, and this same secret is used to decode 
+the token in the receiver side. This token is added to the `Authorization` header.
 - `API_KEY`: where an API key is added directly to the `X-Api-Key` header.
 - `NONE`: where no authorization is used.
 
-### Environment variables
-
-Secrets are passed to the Anchor Platform (and Anchor Reference Server) via environment variables, which can be set either through command line or using a `.env` file. A list of supported environment variables and their descriptions can be found at [`example.env`].
+The following `auth` secrets are required:
+```text
+# REQUIRED - JWT secrets used to communicate between Anchor and Platform.
+SECRET_CALLBACK_API_AUTH_SECRET=<secret>
+SECRET_PLATFORM_API_AUTH_SECRET=<secret>
+```
 
 ### Supported Assets
 
-The Anchor Platform reads the list of supported assets from a json file whose address is configured in the config file under `app-config.app-assets` and defaults to [`assets-test.json`] ([ref](https://github.com/stellar/java-stellar-anchor-sdk/blob/1f84429f0c5d35cee75445686242643fbd8cffa5/platform/src/main/resources/anchor-config-defaults.yaml#L74)).
+The Anchor Platform reads the list of supported assets (to be set in the configuration `override` file) in json form, 
+an example can be found in ['example.anchor-config.yaml'] under `assets`. 
```

Comment:
`['example.anchor-config.yaml'] isn't matching any link added to the bottom of the file